Xiaomi/RedmiNote7/res/values/notch.xml

```diff
@@ -18,8 +18,8 @@
 -->
 <resources>
     <!-- Height of the status bar in portrait -->
-    <dimen name="status_bar_height_portrait">89px</dimen>
```

Comment:
Uh nope, px are better than do (so that it works when user changes dpi)

Reply:
no concerns... not looked so far.